other/upgrade_2-1_mysql.sql

```diff
@@ -51,6 +51,13 @@ CREATE TABLE IF NOT EXISTS {$db_prefix}member_logins (
 ) ENGINE=MyISAM;
 ---#
 
+---# SMF 2.1 requires registration and policy agreements to be enabled so they must be set to 1.
+REPLACE INTO {$db_prefix}settings
+	(variable, value)
+VALUES ('requireAgreement', '1'),
+	   ('requirePolicyAgreement', '1');
+---#
+
```

Comment:
If you do this, they run upgrade script once, it adds it.  If they change the setting then run upgrade again, this will revert it.  We should only insert it if it doesn't exist.

If these are required and shouldn't be disabled, then it should be removed as a setting.